contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -386,7 +395,24 @@ var ChannelModel = BaseModel.extend({
     get_root:function(tree_name){
     	return new ContentNodeModel(this.get(tree_name));
     },
-
+    initialize: function () {
+    	if (this.get("thumbnail_encoding") && typeof this.get("thumbnail_encoding") !== "object"){
```

Comment:
This if statement seems to be repeated a lot on all these functions. Can you find a way to make it DRY?